runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixOverlay.java

```diff
@@ -39,14 +39,14 @@
 
 class TearsOfGuthixOverlay extends Overlay
 {
-	private static final Color CYAN_ALPHA = new Color(Color.CYAN.getRed(), Color.CYAN.getGreen(), Color.CYAN.getBlue(), 100);

```

Comment:
These two removed colors should be what the defaults are.

Reply:
Oops, thanks for that. I've pushed up a change to resolve.